XenAdmin/Wizards/GenericPages/SelectMultipleVMDestinationPage.cs

```diff
@@ -383,6 +383,9 @@ private void PopulateDataGridView(List<ReasoningFilter> homeserverFilters)
                 {
                     var tb = new DataGridViewTextBoxCell {Value = kvp.Value.VmNameLabel, Tag = kvp.Key};
                     var cb = new DataGridViewEnableableComboBoxCell{FlatStyle = FlatStyle.Flat};
+                    List<ReasoningFilter> homeserverFilters = CreateTargetServerFilterList(

```

Comment:
This is the only place where `CreateTargetServerFilterList`, so using the new overload here breaks the Import wizard case, where we only have an override for the old function (CreateTargetServerFilterList(IEnableableXenObjectComboBoxItem item). See ImportSelectHostPage.cs.
So by using the new overload, we will always have an empty `List<ReasoningFilter>` in the Import Wizard, whne we should have the

Reply:
Very good point!
I will refine here. Thanks!